modules/administration-guide/partials/con_supported-restic-compatible-backup-servers.adoc

```diff
@@ -0,0 +1,20 @@
+[id="supported-restic-compatible-backup-servers_{context}"]
+= Supported Restic-compatible backup servers
+
+{prod-short} uses {prod-short} Operator and integrated Restic for backing up and restoring {prod-short} instances from backup snapshots on a configured backup server. {prod-short} Operator automates the creation of a Restic backup repository on the backup server. To back up, {prod-short} Operator gathers the data needed for a backup snapshot and uses the integrated Restic to create and manage the snapshot. To restore, {prod-short} Operator uses the integrated Restic to retrieve and decrypt the snapshot, and then {prod-short} Operator applies the retrieved data to perform the recovery.
```

Comment:
```suggestion
{prod-short} uses the {prod-short} Operator and integrated Restic to back up and restore {prod-short} instances from backup snapshots on a configured backup server. The {prod-short} Operator automates the creation of a Restic backup repository on the backup server. To back up data, the {prod-short} Operator gathers the data required for a backup snapshot and uses Restic to create and manage the snapshot. To restore data, the {prod-short} Operator uses Restic to retrieve and decrypt the snapshot, and then the {prod-short} Operator applies the retrieved data to perform the recovery.
```
- `Che Operator` needs a definite article `the Che Operator`
- You don't need to keep repeating `integrated Restic` after stating it once in the first sentence.

Reply:
@bburt-rh Thank you! Fixed.